src/pages/graphql/schema/customer/mutations/confirm-email.md

```diff
@@ -0,0 +1,64 @@
+---
+title: confirmEmail mutation | Commerce Web APIs
+---
+
```

Comment:
Please add the following lines before the head1. 

import BetaNote from '/src/_includes/graphql/notes/beta.md'

<BetaNote />